int32 id
time stamp
geometry_msgs/Pose2D pose
sensor_msgs/LaserScan scan
sensor_msgs/PointCloud2 point_cloud